omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/visit/RetrospectiveVisitFragmentControllerTest.java

```diff
@@ -135,7 +134,7 @@ public void test_shouldReturnConflictingVisits() throws Exception {
 
         when(ui.format(any())).thenReturn("someDate");
 
-        List<SimpleObject> result = (List<SimpleObject>) controller.create(adtService, patient, location, startDate, null, request, ui);
```

Comment:
Why replace `null` with `stopDate`?

Reply:
This method is to catch any overlaps which was automatically parsing any null endDate to startDate which this enhancement altered and thus for it to pass and still be logical it works only if there is an assigned endDate  since i enclosed it in an If condition on line 39 in RetrospectiveVisitFragmentController